source/utilities.tex

```diff
@@ -2047,39 +2040,44 @@
 \indexlibrary{\idxcode{tuple_cat}}
 \begin{itemdecl}
 template <class... Tuples>
-  constexpr tuple<@\placeholder{CTypes}@...> tuple_cat(Tuples&&... tpls);
+  constexpr tuple<CTypes...> tuple_cat(Tuples&&... tpls);
 \end{itemdecl}
 
 \begin{itemdescr}
 \pnum
-In the following paragraphs, let $\tcode{T}_i$ be the $i^{th}$ type in \tcode{Tuples},
-$\tcode{U}_i$ be \tcode{remove_reference_t<Ti>}, and $tp_i$ be the $i^{th}$
+In the following paragraphs, let $\tcode{T}_i$ be the $i^\text{th}$ type in \tcode{Tuples},
+$\tcode{U}_i$ be \tcode{remove_reference_t<T$_i$>}, and $\tcode{tp}_i$ be the $i^\text{th}$
 parameter in the function parameter pack \tcode{tpls}, where all indexing is
 zero-based.
 
 \pnum
 \requires For all $i$, $\tcode{U}_i$ shall be the type
-$\cv_i$ \tcode{tuple<$Args_i...$>}, where $\cv_i$ is the (possibly empty) $i^{th}$
-cv-qualifier-seq and $Args_i$ is the parameter pack representing the element
-types in $\tcode{U}_i$. Let ${A_{ik}}$ be the ${k_i}^{th}$ type in $Args_i$. For all
-$A_{ik}$ the following requirements shall be satisfied: If $\tcode{T}_i$ is
-deduced as an lvalue reference type, then
-\tcode{is_constructible_v<$A_{ik}$, $cv_i$ $A_{ik}$\&> == true}, otherwise
-\tcode{is_constructible_v<$A_{ik}$, $cv_i A_{ik}$\&\&> == true}.
+$\cv_i$ \tcode{tuple<$\tcode{Args}_i$...>}, where $\cv_i$ is the (possibly empty) $i^\text{th}$
+cv-qualifier-seq and $\tcode{Args}_i$ is the parameter pack representing the element
```

Comment:
... but that doesn't make sense, since there is no such grammar element here. Perhaps it means that `$\cv_i$ are the cv-qualifiers of $\tcode{T}_i$`, but I'm not really sure. Let's make the formatting fix to italicize the grammar term cv-qualifier-seq here and file an LWG issue asking for this description to be revisited.